templates/blog-post.html

```diff
@@ -15,7 +15,9 @@
     <p>{{ breadcrumbs(this) }}</p>
     <h1>{{ this.title }}</h1>
     <p>{{ t_posted_by }}
-        {% if this.twitter_handle %}
+        {% if this.mastodon_handle %}
+        <a href="https://mastodon.social/{{ this.mastodon_handle }}">{{ this.author or this.mastodon_handle }}</a>
```

Comment:
This works, but it leads to the link proving through mastodon.social. 
```suggestion
        <a href="https://{{ this.mastodon_handle.split('@')[2] }}/@{{ this.mastodon_handle.split('@')[1] }}">{{ this.author or this.mastodon_handle }}</a>
```